pgpool.docker/entrypoint.sh

```diff
@@ -167,6 +167,7 @@ function generate_certs() {
     echo -e "\n" >> ${PGPOOL_INSTALL_DIR}/etc/pgpool.conf
     echo "ssl_key = '${PGPOOL_INSTALL_DIR}/tls/tls.key'" >> ${PGPOOL_INSTALL_DIR}/etc/pgpool.conf
     echo "ssl_cert = '${PGPOOL_INSTALL_DIR}/tls/tls.crt'" >> ${PGPOOL_INSTALL_DIR}/etc/pgpool.conf
+    echo "ssl_ca_cert = '${PGPOOL_INSTALL_DIR}/tls/root.crt'" >> ${PGPOOL_INSTALL_DIR}/etc/pgpool.conf
```

Comment:
@MatteoGioioso Sorry for the late response.
If `root.crt` doesn't exist, you should skip this step.

```
if [[ -f "${PGPOOL_INSTALL_DIR}/tls/root.crt" ]]; then
    echo "ssl_ca_cert = '${PGPOOL_INSTALL_DIR}/tls/root.crt'" >> ${PGPOOL_INSTALL_DIR}/etc/pgpool.conf
fi
```

If user doesn't provide their own certs, this docker image will generate a self-signed certificate.
Could you add the commands to generate the root CA in https://github.com/pgpool/pgpool2_on_k8s/blob/125d4c6118bf960bacdeb6d1b3bbc6e3ed600ae1/pgpool.docker/entrypoint.sh#L153

Reply:
@pengbo0328 hey, so sorry for the long wait.

> If root.crt doesn't exist, you should skip this step.

Yes, I will do

> Could you add the commands to generate the root CA in

Correct me if I am wrong, but the CA is needed only in case PostgreSQL has certificates as well and we want to `verify-ca` or `verify-full` from pgpool.

Reply:
@pengbo0328 Hello, any update on this? Thanks

Reply:
@MatteoGioioso Sorry for the late response.

> Correct me if I am wrong, but the CA is needed only in case PostgreSQL has certificates as well and we want to verify-ca or verify-full from pgpool. If the user does not provide its own certs then we should just generate the self-signed pairs without the CA;

Yes. You are correct.
Let's generate the self-signed pairs without the CA if users don't provide their own certs.